src/index.js

```diff
@@ -37,48 +36,61 @@ const BUTTONS_ARRAY = [
     },
     {
         value: "*",
-        characters: ""
     },
     {
         value: 0,
         characters: "+"
     },
     {
         value: "#",
-        characters: ""
     },
     {
         value: "Call",
-        characters: ""
     },
     {
         value: "X",
-        characters: ""
     }
 ]
 
     ; (() => {
-
         // HEADER
         const headerElement = document.querySelector("header")
         const addNumberElement = document.querySelector("p")
         addNumberElement.innerText = "Add Number"
 
+        const handleDisplayElement = () => {
+            const displayElement = document.getElementById("display")
+            console.log(displayElement)
+        }
+
         // MAIN
+
         const mainElement = document.querySelector("main")
+        const numbersToDisplay = []
 
         BUTTONS_ARRAY.forEach((button, index) => {
+            const handleDialButtonOnClick = () => {
```

Comment:
Also, there is no need to declare this function separately.

Instead, you could do:

```js
element.onclick = () => {
  // HERE IS YOUR CODE
}
```